lib/Client.js

```diff
@@ -603,8 +603,8 @@ class Client extends EventEmitter {
                 };
             } else if(content.content !== undefined && typeof content.content !== "string") {
                 content.content = "" + content.content;
-            } else if(content.content === undefined && !content.embed && !content.embeds && !file) {
-                return Promise.reject(new Error("No content, file, or embeds"));
+            } else if(content.content === undefined && !content.embed && !content.embeds && !file && (content.stickerIDS === undefined || content.stickerIDS.length === 0)) {
```

Comment:
Why do you check twice that if all the others + stickers exists? Couldn't you just merge sticker check into the same if closure?

Reply:
sir the red highlighted was removed and the green was added